WordPress/src/main/res/values/strings.xml

```diff
@@ -1225,8 +1225,8 @@
     <string name="error_media_load">Unable to load media</string>
     <string name="error_media_save">Unable to save media</string>
     <string name="error_media_canceled">Uploading media were canceled</string>
-    <string name="error_media_recover_post">We were unable to upload this post\'s media. Please edit the post to try again.</string>
-    <string name="error_media_recover_page">We were unable to upload this page\'s media. Please edit the page to try again.</string>
+    <string name="error_media_recover_post">We were unable to upload this post\'s media. Please tap retry or edit the post to try again.</string>
```

Comment:
This is really wordy, plus it uses the pronoun "we" which Google [recommends avoiding](https://material.io/guidelines/style/writing.html#writing-language). The user should be able to figure out they can edit or retry without us telling them, so how about simply, "Unable to upload this post\'s media?"

Reply:
I'm amazed about how you are so much into details. Thanks! Made the change as suggested in 8cba571